common/lib/xmodule/xmodule/course_module.py

```diff
@@ -197,10 +197,11 @@ class CourseFields(object):
         scope=Scope.settings,
     )
     advertised_start = String(
-        display_name=_("Course Advertised Start Date"),
+        display_name=_("Course Advertised Start"),
         help=_(
-            "Enter the date you want to advertise as the course start date, if this date is different from the set "
-            "start date. To advertise the set start date, enter null."
+            # pylint: disable=C0301
```

Comment:
Instead of quieting pylint here, fix the quality issue and reduce the line length.